src/onlyspans.polyleads.ui/src/components/DocumentsManager/documents/DocumentEditingModal/DocumentEditingModal.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import {
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from '@/components/ui/dialog';
+import { Button } from '@/components/ui/button';
+import { Label } from '@/components/ui/label';
+import { Input } from '@/components/ui/input';
+import useGet from '@/hooks/useGet';
+import ServiceSymbols from '@/data/constant/ServiceSymbols';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormMessage,
+} from '@/components/ui/form';
+import { IDocumentEditingModalVM } from '@/components/DocumentsManager/documents/DocumentEditingModal/DocumentEditingModal.vm';
+import { IDocument } from '@/data/abstractions/IDocument';
+
+interface IDocumentEditingModal {
+  document: IDocument;
+}
+
+const DocumentEditingModal: React.FC<IDocumentEditingModal> = ({document}) => {
```

Comment:
```suggestion
const EditDocumentModal: React.FC<IDocumentEditingModal> = ({document}) => {
```

в остальных местах так же